### ISSUE TITLE
Disable dismiss cookie consent dialog on outside click 

### ISSUE THREAD

**Issue by deeppandya** [brave/brave-browser#25811]
This issue is to disable dismiss cookie consent dialog on outside click. 
For more context : https://bravesoftware.slack.com/archives/CKNHM4RTK/p1664405692434979

**Comment by kjozwiak** [comment]
The above requires `1.45.104` or higher for `1.45.x` verification 👍

**Comment by kjozwiak** [comment]
Labelling as `QA/Blocked` as we'll need to wait for https://github.com/brave/brave-browser/issues/26078 to be fixed and uplifted into `1.45.x`. We'll also need to enable `BraveAdblockCookieListOptInStudy` on `BETA` and `Release` before we can verify the above as we'll be enabling this feature via Grffin on both `Desktop` & `Android`.

@Uni-verse @srirambv I can run through this as I have the most context/been working on this for a while.

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Samsung S10+` running `Android 12` using the following build(s):

```
Brave | 1.45.111 Chromium: 107.0.5304.62 (Official Build) (32-bit)
--- | ---
Revision | 1eec40d3a5764881c92085aaee66d25075c159aa-refs/branch-heads/5304@{#942}
OS | Android 12; Build/SP1A.210812.016
```

Verification was completed via https://github.com/brave/brave-browser/issues/8974#issuecomment-1289062618
